worlds/stardew_valley/items.py

```diff
@@ -151,58 +151,58 @@ def create_items(item_factory: StardewItemFactory, locations_count: int, items_t
-    create_tools(item_factory, world_options, items)
-    create_skills(item_factory, world_options, items)
-    create_wizard_buildings(item_factory, world_options, items)
-    create_carpenter_buildings(item_factory, world_options, items)
+    create_elevators(item_factory, options, items)
+    create_tools(item_factory, options, items)
+    create_skills(item_factory, options, items)
+    create_wizard_buildings(item_factory, options, items)
+    create_carpenter_buildings(item_factory, options, items)
     items.append(item_factory("Beach Bridge"))
     items.append(item_factory("Dark Talisman"))
     create_tv_channels(item_factory, items)
     create_special_quest_rewards(item_factory, items)
-    create_stardrops(item_factory, world_options, items)
-    create_museum_items(item_factory, world_options, items)
-    create_arcade_machine_items(item_factory, world_options, items)
+    create_stardrops(item_factory, options, items)
+    create_museum_items(item_factory, options, items)
+    create_arcade_machine_items(item_factory, options, items)
     items.append(item_factory(random.choice(items_by_group[Group.GALAXY_WEAPONS])))
-    create_player_buffs(item_factory, world_options, items)
+    create_player_buffs(item_factory, options, items)
     create_traveling_merchant_items(item_factory, items)
     items.append(item_factory("Return Scepter"))
-    create_seasons(item_factory, world_options, items)
-    create_seeds(item_factory, world_options, items)
-    create_friendsanity_items(item_factory, world_options, items)
-    create_festival_rewards(item_factory, world_options, items)
+    create_seasons(item_factory, options, items)
+    create_seeds(item_factory, options, items)
+    create_friendsanity_items(item_factory, options, items)
+    create_festival_rewards(item_factory, options, items)
     create_babies(item_factory, items, random)
-    create_special_order_board_rewards(item_factory, world_options, items)
-    create_special_order_qi_rewards(item_factory, world_options, items)
-    create_walnut_purchase_rewards(item_factory, world_options, items)
-    create_magic_mod_spells(item_factory, world_options, items)
+    create_special_order_board_rewards(item_factory, options, items)
+    create_special_order_qi_rewards(item_factory, options, items)
+    create_walnut_purchase_rewards(item_factory, options, items)
+    create_magic_mod_spells(item_factory, options, items)
 
     return items
 
 
-def create_backpack_items(item_factory: StardewItemFactory, world_options: StardewOptions, items: List[Item]):
-    if (world_options[options.BackpackProgression] == options.BackpackProgression.option_progressive or
-            world_options[options.BackpackProgression] == options.BackpackProgression.option_early_progressive):
+def create_backpack_items(item_factory: StardewItemFactory, options: StardewValleyOptions, items: List[Item]):
+    if (options.backpack_progression == BackpackProgression.option_progressive or
```

Comment:
```suggestion
    if options.backpack_progression in {BackpackProgression.option_progressive, BackPackProgression.option_early_progressive}:```

Reply:
This code is changed in my WIP branches anyway, and it's out of scope for this